LinksAwakeningClient.py

```diff
@@ -404,6 +410,10 @@ async def main_tick(self, item_get_cb, win_cb, deathlink_cb):
         if await self.is_victory():
             await win_cb()
 
+        today = datetime.date.today()

```

Comment:
I believe getting the current date on every tick is considered expensive.

Reply:
It shouldn't be _that_ expensive - but you're right, I can check on open or every hour or something.